app/(entrance)/sign-up/components/SignUpPage.js

```diff
@@ -1,8 +1,9 @@
 'use client'
 import { isValidEmail } from '@shared/inputs/EmailInput.js'
+import { isValidPhone } from '@shared/inputs/PhoneInput.js'
 import PasswordInput from '@shared/inputs/PasswrodInput.js'
```

Comment:
This isn't part of your changes, but it'd be so nice if the `PasswrodInput.js` -> `PasswordInput.js` filename fix could be done since we're making changes in the neighborhood.